sdk/keyvault/azure-keyvault-keys/tests/test_key_client.py

```diff
@@ -359,3 +362,73 @@ def test_purge(self, vault_client, **kwargs):
         # validate none are returned by list_deleted_keys
         deleted = [s.name for s in client.list_deleted_keys()]
         self.assertTrue(not any(s in deleted for s in key_names))
+
+    @ResourceGroupPreparer(name_prefix=name_prefix)
+    @VaultClientPreparer(client_kwargs={'logging_enable': True})
+    def test_logging_enabled(self, vault_client, **kwargs):
+        class MockHandler(logging.Handler):
+            def __init__(self):
+                super(MockHandler, self).__init__()
+                self.messages = []
+            def reset(self):
+                self.messages = []
+            def emit(self, record):
+                self.messages.append(record)
+        self.assertIsNotNone(vault_client)
+        client = vault_client.keys
+        mock_handler = MockHandler()
+
+        # Configure a console output
```

Comment:
Not what we're doing here 😁

Reply:
i should be more careful when i'm copy-pasting